.travis.yml

```diff
@@ -3,24 +3,26 @@ cache: packages
 sudo: true
 dist: trusty
 
-addons:
-  apt:
-    packages:
-      - libfreetype6
-
-r:
-- 3.1
-- 3.2
-- oldrel
-- release
-- devel
+# build matrix; skip vdiffr on all but r release
+matrix:
+  include:
+  - r: 3.1
+    env: SKIP_VDIFFR=true
+  - r: 3.2
+    env: SKIP_VDIFFR=true
+  - r: oldrel
+    env: SKIP_VDIFFR=true
+  - r: release
```

Comment:
Since you're in here, would you mind re-organsing so the versions are in reverse chronological order? That way the most recent versions (which are mostly likely to reveal problems) will be run first

Reply:
Done.